swift_scality_backend/diskfile.py

```diff
@@ -93,17 +96,25 @@ def __init__(self, conf, logger):
         if conf_wants_splice and system_has_splice:
             self.use_splice = True
 
+        timeout = urllib3.Timeout(connect=self.conn_timeout,
+                                  read=self.proxy_timeout)
+        # One HTTP Connection pool per sproxyd host
+        self.http_pools = urllib3.PoolManager(len(self.sproxyd_hosts_set),
+                                              timeout=timeout, retries=False,
+                                              maxsize=32)
```

Comment:
maxsize is per connection pool, so it's not max 32 connections spread among all connection pools but max 32 connection per pool.